### ISSUE TITLE
Building on Linux

### ISSUE THREAD

**Issue by kamunds** [creativetimofficial/ct-vue-now-ui-dashboard-pro#3]
Following your install instructions in my Debian Linux environment and am getting some dependency errors:

UNMET PEER DEPENDENCY popper.js@^1.12.9

npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.3
npm WARN optional Skipping failed optional dependency /watchpack/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.3
npm WARN optional Skipping failed optional dependency /webpack-dev-server/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.3

npm WARN bootstrap@4.0.0 requires a peer of popper.js@^1.12.9 but none was installed.

From what I can tell, 'fsevents' is a Mac-only package.  Does this mean that I need a Mac to develop using your Dashboard?

Also, the 'popper.js' errors appear to be related to Bootstrap 4.  Are you aware of this dependency issue?

Thanks!

**Comment by cristijora** [comment]
Hi @kamunds Really sorry for the late response. Somehow, this issue slipped through my github notifications. Is this the full log you get ? Regarding Mac-only package, the dashboard should work on any platform. I've tested on Mac and Windows for now. 

My assumption here is that since bootstrap comes with the javascript part as well, it requires popper.js as well because in the dependency list we have `jquery` which is used for fullcalendar and vector maps. Even if jquery is not used along with bootstrap, npm sees that jquery and bootstrap are both listed in package.json and tries to make sure popper.js is installed as well.

So long story short I have 2 suggestions:
- If you plan to use full calendar or vector maps, install `popper.js`.
- If you don't need those 2 components from above, uninstall jquery.

Also do you mind sharing the npm/node version you have on linux?
It seems that on linux these are treated as errors while on mac/windows as warnings.

**Comment by cristijora** [comment]
Closing due to inactivity
